src/VisualStudio/Core/Def/Implementation/Venus/ContainedDocument.cs

```diff
@@ -234,6 +244,14 @@ public string FilePath
         {
             get
             {
+                if (_hostType == HostType.TypeScript)
+                {
+                    // Append a .js extension to the path 
+                    // so that the contained document will  
+                    // be processed as JavaScript.
+                    return Key.Moniker + JSFileExtension;
```

Comment:
The TypeScript language service now powers both TypeScript and JavaScript editing. The language service feature experiences are often different depending on the type of file that is being processed. This change is to ensure the contained language script blocks (.html, .cshtml, etc) are always processed as JavaScript code. 
@CyrusNajmabadi: is there another way you'd prefer we handle this?